tests/std/include/range_algorithm_support.hpp

```diff
@@ -610,6 +593,7 @@ struct with_writable_iterators {
     template <class... Args>
     static constexpr void call() {
         using namespace test;
+        using test::iterator; // Hello, one-phase legacy name lookup!

```

Comment:
I'm not sure what this comment means (admittedly it is midnight and my brain is running out of caffeine).

Reply:
I had a compile error remind me that using-directives move backward in time in MSVC's permissive mode. I'm also trying to `using namespace std` in new test code (ditto overall consistency of the test codebase - `using namespace std` is the norm) and `using namespace std` _after_ including this header affects unqualified name lookup _within_ this header. `using test::iterator` is necessary to disambiguate between `test::iterator` and `std::iterator`.

The comment is more about my horrified reaction than contributing any useful information to the reader, I'll strike it.

Reply:
Do you suggest preserving the memory of your horrified reaction is not worth it?

Reply:
I'm confident that the nightmares will remain even after the comment is removed. ;)